CHANGELOG.md

```diff
@@ -42,6 +42,12 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * Update the swagger files (including third-party changes). [#728](https://github.com/provenance-io/provenance/issues/728)
 * Bump IBC to 2.3.0 and update third-party protos [PR 868](https://github.com/provenance-io/provenance/pull/868)
 
+### Features
+
+* Create authz authorization with limited number of uses. [#807](https://github.com/provenance-io/provenance/issues/807)
+  * Create a new authz authorization type that is limited to a certain number of times [#904](https://github.com/provenance-io/provenance/issues/904)
+  * Update metadata and marker modules to delete authorizations based on Delete flag [#728](https://github.com/provenance-io/provenance/issues/728)
```

Comment:
The first two of these entries are the same, just with different issue numbers. You can get rid of one of them. Also, it'd be better and more accurate to phrase it with reference to the bumped cosmos-sdk version since that's where that new authz changes were actually made. Furthermore, identifying the new authorization type in here would help readers since the first question they're going to ask is, "What's this new type of authorization?"

```suggestion
* Bump Cosmos-SDK to `v0.45.4-pio-4` (from `v0.45.4-pio-2`) to utilize the new `CountAuthorization` authz grant type. [#807](https://github.com/provenance-io/provenance/issues/807)
* Update metadata module authz handling to properly call `Accept` and delete/update authorizations as they're used [#728](https://github.com/provenance-io/provenance/issues/728)
```